bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -6,10 +6,12 @@ Furthermore it is possible to embed `command tags` in the calendar event descrip
 
 ## Supported Things
 
-The only thing type is the calendar.
-It is based on a single iCalendar file.
+The primary thing type is the calendar.
+It is based on a single iCalendar file and implemented as bridge.
 There can be multiple things having different properties representing different calendars.
 
+Each calendar can have event filters which allow to get multiple events, maybe filtered by additional criteria. Time based filtering is done by each event's start.
+
 ## Thing Configuration
 
 Each `calendar` thing requires the following configuration parameters:
```

Comment:
I suggest to have a sub title ###Thing Configuration for "calendar" here

Reply:
OK.